### PR TITLE
HDDS-7579. Use Netty 4.1.77 for consistency

### PR DESCRIPTION
**Pull request by adoroszlai**
## What changes were proposed in this pull request?

Ratis uses Netty 4.1.77.  The same version is recommended for gRPC 1.48.1 which Ozone uses.  This PR proposes to move from Netty 4.1.79 to Netty 4.1.77 for consistency.

https://issues.apache.org/jira/browse/HDDS-7579

## How was this patch tested?

Regular CI:
https://github.com/adoroszlai/hadoop-ozone/actions/runs/3600140354

### CODE REVIEW

**Review by kerneltime — APPROVED** [review]

**Review by szetszwo — APPROVED** [review]
+1 the change looks good.

**Review by aswinshakil — APPROVED** [review]

File: pom.xml [hunk 1]
```diff
@@ -186,7 +186,7 @@ xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xs
     <grpc.protobuf-compile.version>3.19.6</grpc.protobuf-compile.version>
     <os-maven-plugin.version>1.5.0.Final</os-maven-plugin.version>
 
-    <netty.version>4.1.79.Final</netty.version>
+    <netty.version>4.1.77.Final</netty.version>
```
Comment (smengcl) [inline pom.xml:189]:
```suggestion
    <!-- Use netty version known to work with grpc-netty. See table: -->
    <!-- https://github.com/grpc/grpc-java/blob/master/SECURITY.md#netty -->
    <netty.version>4.1.77.Final</netty.version>
```

**Review by smengcl — APPROVED** [review]
Good catch @adoroszlai !

I guess we can just add a line of comment (or two) in the pom.xml itself so others maintaining this are also informed.

**Comment by smengcl** [comment]
Thanks @adoroszlai for the patch. @kerneltime @szetszwo @aswinshakil for reviewing.

**Pull request merged**